### PR TITLE
Modify backup intent condition

### PR DESCRIPTION
**Pull request by nakulkar-msft**

### CODE REVIEW

File: ste/sender-azureFileFromURL.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func (u *urlToAzureFileCopier) GenerateCopyFunc(id common.ChunkID, blockIndex in
 			return
 		}
 		ctx := u.ctx
-		if u.addFileRequestIntent || (token != nil && u.jptm.FromTo().From() == common.ELocation.File()) {
+		if u.addFileRequestIntent {
```
Comment (gapra-msft) [inline ste/sender-azureFileFromURL.go:75]: why are we removing this check?

Comment (nakulkar-msft) [reply ste/sender-azureFileFromURL.go:75]: The rest documentation does not require this header when the source is auth'd via a token.

Comment (gapra-msft) [reply ste/sender-azureFileFromURL.go:75]: Can you double check we have a test where source is Authed with Oauth and destination uses SAS for file to file copy? I think that is the scenario I hit when I needed to add it

**Comment by siminsavani-msft** [comment]
@nakulkar-msft Could you please add a description about the change?

**Pull request closed without merging**